app/models/P5/departure/ConsignmentItem.scala

```diff
@@ -19,11 +19,61 @@ package models.P5.departure
 import play.api.libs.json.Json
 import play.api.libs.json.OFormat
 
-case class ConsignmentItem(Packaging: Seq[Packaging]) {
+case class ConsignmentItem(
+  declarationType: Option[String],
+  countryOfDispatch: Option[String],
+  countryOfDestination: Option[String],
+  Consignor: Option[Consignor],
+  Consignee: Option[Consignee],
+  goodsItemNumber: String,
+  declarationGoodsItemNumber: Int,
+  Packaging: Seq[Packaging],
+  Commodity: Commodity,
+  referenceNumberUCR: Option[String],
+  TransportCharges: Option[TransportCharges],
+  PreviousDocument: Option[List[PreviousDocument]],
+  SupportingDocument: Option[List[SupportingDocument]],
+  TransportDocument: Option[List[TransportDocument]],
+  AdditionalReference: Option[List[AdditionalReference]],
+  AdditionalInformation: Option[List[AdditionalInformation]],
+  AdditionalSupplyChainActor: Option[List[AdditionalSupplyChainActor]],
+  DepartureTransportMeans: Option[List[DepartureTransportMeans]]
```

Comment:
I don't see this at the consignment item level in the IE029 in the TIS 🤔

Reply:
same as above consignment, now copied from consignment level to item level.